src/request.ts

```diff
@@ -452,19 +466,24 @@ class DatastoreRequest {
    *   const entities = data[0];
    * });
    */
-  get(keys, options?): Promise<EntityDataObj[]>;
-  get(keys, options?, callback?): void|Promise<EntityDataObj[]> {
-    if (is.fn(options)) {
-      callback = options;
-      options = {};
-    }
-
-    options = options || {};
+  get(keys: Entities,
```

Comment:
Returning the Entity or a Stream doesn't feel quite right.  What am I missin here?

Reply:
I admit the reasoning isn't 100% here. `get` relies on `createReadStream` which returns a stream, but `get` is expecting something iterator-related:
```
Type 'Stream' must have a '[Symbol.iterator]()' method that returns an iterator.

826       const [[deletedEntity], [fetchedEntity]] = await Promise.all([
```
Changing it to `Promise<Entity>` works well - lmk your take on it :)